src/Features/CSharp/Portable/AddStaticModifier/CSharpAddStaticModifierCodeFixProvider.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using Microsoft.CodeAnalysis.AddStaticModifier;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.CodeAnalysis.CSharp.AddStaticModifier
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
```

Comment:
I am not sure whether or not I should provide the `Name =...` part here. One of the 2 examples have it the other doesn't.

Reply:
Yes, set the name to `nameof(CSharpAddStaticModifierCodeFixProvider)`